ASIM/dev/Parser YAML templates/vimUserManagementTemplate.yaml

```diff
@@ -0,0 +1,41 @@
+Parser:
+  Title: User Management activity ASIM parser for <product name>
+  Version: '<parser version>'
+  LastUpdated: <parser update date>
+Product:
+  Name: <product name>
+Normalization:
+  Schema: UserManagement
+  Version: '<current schema version>'
+References:
+- Title: ASIM User Management Schema
+  Link: https://aka.ms/ASimUserManagementDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing the <product name> logs to the ASIM User Management activity normalized schema.
+ParserName: <parser function name>
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: disabled
+    Type: bool
+    Default: false
```

Comment:
missing filters. Please refer:
https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimUserManagement/Parsers/vimUserManagementMicrosoftSecurityEvent.yaml

Reply:
@azurekid I need to update ASIM documentation